<launch>
	<node pkg="turtlesim" type="turtle_teleop_key" name="turtle_teleop_key" output="screen" launch-prefix="gnome-terminal --command" />	

	<node pkg="balboa_core" type="balboa_serial.py" name="balboa_serial" />

        <node pkg="lab1" type="pidLab1.py" name="pidLab1" output="screen">

		<param name="pValAng" value="0.0001" />
		<param name="dValAng" value="0.00000001" />
		<param name="pValDis" value="0.01" />
		<param name="dValDis" value="0.0005" />
	</node>

</launch>
